src/store/index.js

```diff
@@ -44,7 +45,8 @@ export default new Vuex.Store({
             registration_form: {},
             contact_form: {}
         },
-        token: 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwOi8vbG9jYWxob3N0OjgwODAvYXBpL3YxL2F1dGgvbG9naW4iLCJpYXQiOjE2NjQ4ODYyNjksImV4cCI6MTY2NDk3MjY2OSwibmJmIjoxNjY0ODg2MjY5LCJqdGkiOiJQTmREdXE1bVBEc2lrSUIzIiwic3ViIjo0MiwicHJ2IjoiMjNiZDVjODk0OWY2MDBhZGIzOWU3MDFjNDAwODcyZGI3YTU5NzZmNyJ9.wJszEQVp-ZQGJFAP_kdnwKe1xOMAR0YwX1HsLLogBk0',
+        
```

Comment:
Was the change here intentional? If you used `state.api.client` then the change would have been inside your `auth middleware`, you can explicit set the token in the header options like:
```
let req_options = {
    headers: {"
        Authorization: 'Bearer' + token
    }
}

this.state.api.client.post(url, {}, req_options)
.then(function (response) {
    //success
}).catch(function (error) {
    //error
})
```

Kindly reset the change in this file.